src/graphql/types/index.ts

```diff
@@ -2,7 +2,7 @@
 import BTCWallet from "./object/btc-wallet"
 import ConsumerAccount from "./object/consumer-account"
 // import FiatWallet from "./object/fiat-wallet"
-import WalletNameTransaction from "./object/wallet-name-transaction"
+import UsernaemTransaction from "./object/wallet-id-transaction"
```

Comment:
It looks like this was changed to [`IntraLedgerTransaction`](https://github.com/GaloyMoney/galoy/pull/602/files#diff-62e0e652988ce1a59d766870da93ce8f1f3817333e2fbaa5d9493ea085efa6d7R13)

```suggestion
import IntraLedgerTransaction from "./object/wallet-id-transaction"
```